src/rgw/rgw_tools.cc

```diff
@@ -451,10 +451,10 @@ int rgw_tools_init(CephContext *cct)
 {
   ext_mime_map = new std::map<std::string, std::string>;
   int ret = ext_mime_map_init(cct, cct->_conf->rgw_mime_types_file.c_str());
-  if (ret < 0)
-    return ret;
-
-  return 0;
+  if (ret < 0) {
```

Comment:
Given that we ignore the rv anyway, I guess we can simply return 0 now after parsing and avoid the whole if block?

Reply:
updated, thanks